WebRTC_backend/textchat/client.js

```diff
@@ -0,0 +1,210 @@
+//our username 
+var name; 
+var connectedUser; 
+
+//connecting to our signaling server 
+var conn = new WebSocket('ws://52.200.7.179:55535');
```

Comment:
We should change this line to `var conn = new WebSocket('ws://localhost:9090');` for the local version